.github/workflows/ci-workflow.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.6.15, 3.7, 3.8]
+        python-version: [3.11]
```

Comment:
Note that since these tests are running independently of the target SLE deployment ecosystem, there is no hard in having them run for multiple viable versions of Python.

However, Python 3.6 and Python 3.7 are not viable within the ubuntu-22.04 and later runners, so the minimum version that you can test for is 3.8.

And per my #145 the package versions in the requirements.txt and/or test-requirements.txt are not compatible with Python 3.12 or later, so this list should be left at 3.8, 3.9, "3.10", and 3.11. Note the quoting around "3.10" because without it the value will be treated as a floating point number by YAML parsing and 3.10 == 3.1.